tests/test_util.py

```diff
@@ -408,6 +408,16 @@ def test_get_manifest_digests_missing(tmpdir, has_content_type_header, has_conte
 
     url = 'https://example.com/v2/spam/manifests/latest'
 
+    def custom_retries(*args, **kwargs):
```

Comment:
This resets the remaining amount of retries to 0, so with a proper backoff formula it should wait the longest amount.

Perhaps replacing `retries` to `1` would be faster